src/ImageBuilder.ts

```diff
@@ -379,8 +394,8 @@ export default class ImageBuilder {
 
     async executeAzCliCommand(command: string): Promise<string> {
         var outStream: string = '';
-        console.log("az cli command " + command);
         var execOptions: any = {
+            outStream: new NullOutstreamStringWritable({ decodeStrings: false }),
```

Comment:
With this change in place, if a cli command fails, are we getting the error message printed in logs?
This is important that we print as much error log as possible from cli commands.

Reply:
if we wont add it then it will print all the logs and it will be too much.So not very sure of this if to remove it or let it be.

Reply:
I understand that. But we also need to make sure we are propagating the right error messages. Can you fail a command and see if the error shows in logs?

Reply:
we get this
$(imagebuilder-run-status) =  failed
##[error]Error: The process '/usr/bin/az' failed with exit code 1
##[error]Action run failed.

Reply:
Yes, this is not good. This doesn't tell what went wrong. 
Can you check some other actions how they are handling this?

Reply:
ohkay